adapters/handlers/rest/clusterapi/db_users.go

```diff
@@ -0,0 +1,91 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package clusterapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/weaviate/weaviate/usecases/auth/authentication/apikey"
+)
+
+type DbUsers struct {
+	userManager *apikey.RemoteApiKey
+	auth        auth
+}
+
+func NewDbUsers(manager *apikey.RemoteApiKey, auth auth) *DbUsers {
+	return &DbUsers{userManager: manager, auth: auth}
+}
+
+func (d *DbUsers) Users() http.Handler {
+	return d.auth.handleFunc(d.userHandler())
+}
+
+func (d *DbUsers) userHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		path := r.URL.Path
+		switch path {
+		case "/cluster/users/db/":
```

Comment:
hmm, shouldn't we here remove the trailing `/` from the end? or include also `"/cluster/users/db"`? or maybe instead add a regexp that will catch both of those urls?

Reply:
I changed the url to "/cluster/users/db/lastUsedTime"